program_intent_engagement/apps/api/v1/views.py

```diff
@@ -112,12 +112,13 @@ def grab_best_intents(cls, user_intents):
         ):
             intents = list(intents)
 
-            # sort by certainty, effective_timestamp
+            # sort by certainty ascending, effective_timestamp descending
             intents.sort(
                 key=lambda x: (
-                    certainty_order[(x["certainty"])],
+                    -certainty_order[(x["certainty"])],
```

Comment:
Another approach here is changing the certainty values but I figured this would be a more readable approach. Reverse sort, then reverse=True.

Reply:
I agree having the certainty value list in order by importance is probably going to generate fewer bugs :)

Reply:
Cool cool!